modules/local/samplesheet_check_screening.nf

```diff
@@ -0,0 +1,27 @@
+process SAMPLESHEET_CHECK_SCREENING {
```

Comment:
Is there no samplesheet check for `targeted`?

Reply:
There is, but we maintained the original name to match the template. I would rather fix that after we implement the nf-validation plugin, which will reduce the amount of code considerably.